test/e2e/getenvoy_run_test.go

```diff
@@ -23,14 +23,16 @@ import (
 	"testing"
 	"time"
 
-	"github.com/mholt/archiver/v3"
 	"github.com/stretchr/testify/require"
+
+	internalreference "github.com/tetratelabs/getenvoy/internal/reference"
+	"github.com/tetratelabs/getenvoy/internal/tar"
 )
 
 const terminateTimeout = 2 * time.Minute
 
 // reference holds the argument to 'getenvoy run'
-var reference string
+var reference = internalreference.Latest
```

Comment:
now I think this variable is redundant imho

Reply:
yep